src/core/utils/BasicHtmlRenderingEngine.ts

```diff
@@ -50,6 +50,12 @@ export const BasicHtmlRenderingEngine = {
         if (!node.hasChildren() && !node.properties.atomic) {
             renderedElement.appendChild(document.createElement('BR'));
         }
+
+        // If a LINE_BREAK has no next sibling, it must be rendered as two BRs
+        // in order for it to be visible.
+        if (node.type === VNodeType.LINE_BREAK && !node.nextSibling()) {
+            fragment.appendChild(document.createElement(tagName));
+        }
```

Comment:
Here is another place where we should have different classes/renderers, such that this only exist in the renderer of a `LINE_BREAK` node such that the type check is not needed.